ruby/src/IceRuby/Config.h

```diff
@@ -79,39 +79,4 @@ extern "C"
 
 #define CAST_METHOD(X) reinterpret_cast<ICE_RUBY_ENTRY_POINT>(X)
 
-//
```

Comment:
We can drop support for these ancient Ruby versions and remove the compatibility macros.